modules/social_features/social_event/modules/social_event_enrollment_max/config/install/field.storage.node.field_event_max_enroll.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
```

Comment:
This module should be an enforced dependency of this storage field so it gets removed when the module is uninstalled.

Also goes for other storage fields and config items which don't have it yet!